profile-service/src/main/resources/config/application.yml

```diff
@@ -176,7 +176,7 @@ jhipster:
             enabled: true
             endpoint: /management/prometheus
         logs: # Reports Dropwizard metrics in the logs
-            enabled: false
+            enabled: true
```

Comment:
Log retention period is 7 days, I guess. As this is now set to true, it will not exceed the limit where AWS starts charging, hopefully.